user_guide.md

```diff
@@ -322,6 +322,10 @@ inception   LoadBalancer   10.35.255.136   ww.xx.yy.zz   9000:30936/TCP   28m
 
 In this example, you should be able to use the inception_client to hit ww.xx.yy.zz:9000
 
+The model at gs://kubeflow-models/inception is public accessible. However, if your environment doesn't
+have google authentication setup, TF serving will have problem reading the model.
```

Comment:
Do we have a link to what it means to have GCP credentials?
"TF serving will not be able to read the model"

Reply:
nit: same as above "publicly"